playbooks/templates/standalone_parameters.yaml.j2

```diff
@@ -22,6 +22,11 @@ parameter_defaults:
   InterfaceLocalMtu: 1500
   SELinuxMode: permissive
   StandaloneNetworkConfigTemplate: "{{ ansible_env.HOME }}/dev-install_net_config.yaml"
+  StandaloneNetConfigOverride:
```

Comment:
I might put a comment or even an if block here (to entirely omit it) to make it really obvious that this is OSP16 only. I wasted nearly 2 days working this out first time! Wouldn't want anybody to do that again just because we're setting it in this file.